lessons/08-html-site/index.html

```diff
@@ -0,0 +1,68 @@
+<html>
+    <head>
+        <title>Selector Style Attributes</title>
+        <style>
+            h1 {
+                text-align: center
+            }
+        </style>
+        <style>
+            .para1,
+            .para2,
+            .para3{
+                text-align: right
+            }
+        </style>
+        <style>
+            .para1{
+                font-style: italic
+            }
+        </style>
+        <style>
+            .para2{
+                background: blue
+            }
+        </style>
+        <style>
+            .para4, .para5, .para6 {
+                text-align: justify
+            }
+        </style>
+        <style>
+            .para5{
+                background: #00fc00
+            }
+        </style>
+        <style>
+            .para5{
+                text-transform: uppercase
+            }
+        </style>
+        <style>
+            .para6{
+                text-transform: capitalize
+            }
+        </style>
```

Comment:
These can all be inside of one `<style>` tag 😄 

e.g.

```html
<style>
.para5{
  text-transform: uppercase;
}
.para6{
  text-transform: capitalize;
}
</style>
```

Reply:
Also please make sure to end each rule with a semi-colon `;`

e.g. `text-transform: uppercase;`